app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -809,8 +869,8 @@ export default class Heatmap {
       .enter()
       .append("g")
       .attr("class", cs.rowLabel)
-      .on("mousein", this.options.onRowLabelMouseIn)
-      .on("mouseout", this.options.onRowLabelMouseOut);
+      .on("mouseover", this.handleRowLabelMouseOver)
```

Comment:
`this.options.onRowLabelMouseIn` and `Out` were not used anywhere. 

I made `handleRowLabelMouseOver` a method of `Heatmap` because the behavior is internal to the `Heatmap` and it is enabled by `shouldSortRows` option.

Reply:
This would be used if the client wants to add behavior. For instance, you would like to add a tooltip for this event, right? That could be done on the client.

Reply:
Yes, no clients were adding behavior with those options. That's why I removed them, to avoid confusion with the new internal mouseover behavior. 

For the tooltip that was in @happyimadesignr 's mock, I'm planning now on adding that now in a follow-up PR.

Reply:
No clients are using those options, and so I removed them to avoid confusion with `handleRowLabelMouseOver`.

For the tooltip as in @happyimadesignr 's mock, I'm planning on adding that in a follow-up PR.